### PR TITLE
Portfolio milestone 04

### PR DESCRIPTION
**Pull request by shahadat3669**
In this milestone in my portfolio website, I finished:

- [x] Used media queries to make sure that my website displays correctly on desktop screens.
- [x] To lay out the elements on the page I used Grid and Flexbox
- [x] Replicated the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +17,11 @@
         <div class="header-inner">
           <nav class="header-nav">
             <a href="/" class="header-logo">Welcome</a>
+            <ul class="nav__list">
+              <li class="nav__item">Portoflio</li>
+              <li class="nav__item">About</li>
+              <li class="nav__item">Contact</li>
```
Comment (Yazino12) [inline index.html]: - [x] Please consider using an `<a>` tag for your navigation buttons 🦋, that way users can easily navigate through your page with a click of a button 📎. **_You will end up with something like this_** 👇🏽

image
**_Please don't forget to style the anchor tags after adding so you stick to the design as much as possible_**

File: index.html [hunk 2]
```diff
@@ -323,113 +246,59 @@ <h3 class="card__title">Skills</h3>
...
-              required
-            ></textarea>
-            <button class="btn btn--primary">Get in touch</button>
+          <h2 class="contact-form__title title--3">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          <form action="https://formspree.io/f/myyvbjla" method="POST" class="contact-form__form">
+            <input type="text" id="contact-form__full-name" placeholder="Full name" name="full-name" maxlength="30" required />
+            <input type="email" placeholder="Email address" id="contact-form__email" name="email" required />
+            <textarea name="message" id="contact-form__message" cols="30" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+            <div class="contact-form__btn-wrapper">
+              <button class="btn btn--primary">Get in touch</button>
```
Comment (Yazino12) [inline index.html:262]: - [x] The form button is **center-aligned for mobile screens** and **left-aligned for desktop screens** 📝, but in your design, it's **left-aligned** for both `mobile` and `desktop` screens. Kindly fix this 👇🏽:

### Your design:
image

### Figma design:
image

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @shahadat3669 🖐🏾,

You've come a long way and you are almost there 👏🏽, but there are still some changes you need to work on to move to the next project 🔨.

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 
- Descriptive Readme ✔
- Good PR summary ✔

# STATUS:  REQUIRED CHANGES ♻♻♻ 
THIS CLOSE
_Check the comments under the review 👇🏽👇🏽👇🏽._

- [x] From now on, try to write better commit messages. The sentences should be capitalized and in the imperative form. Example: "Fix background" and not "fixing background" 👍🏽.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shahadat3669** [comment]
- [x] Fixed issues with the nav item 

image

- [x] Fixed contact  alignment
image

**Review by aimalamiri — APPROVED** [review]
Hi @shahadat3669, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**